src/rgw/rgw_http_client.cc

```diff
@@ -121,7 +121,7 @@ int RGWHTTPClient::process(const char *method, const char *url)
 
   CURLcode status = curl_easy_perform(curl_handle);
   if (status) {
-    dout(0) << "curl_easy_perform returned error: " << error_buf << dendl;
+    dout(0) << "curl_easy_perform returned status " << status << " error: " << error_buf << dendl;
```

Comment:
Yeah I'm seeing couldn't connect 7 https://curl.haxx.se/libcurl/c/libcurl-errors.html , we could return here itself, but then since this is commonly used by all other http requests, for eg. keystone connections etc. I don't know the implications fully yet